tests/tests_release_notes_workflow/test_component_release_notes.py

```diff
@@ -0,0 +1,81 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+import unittest
+from typing import Any
+from unittest.mock import patch
+from xmlrpc.client import boolean
+
+from git.git_repository import GitRepository
+from manifests.input_manifest import InputComponentFromSource
+from release_notes_workflow.component_release_notes import ComponentReleaseNotesOpenSearch, ComponentReleaseNotesOpenSearchPlugin, ComponentsReleaseNotes
+
+
+class TestComponentReleaseNotes(unittest.TestCase):
+
+    @patch('subprocess.check_call', return_value=0)
+    @patch('subprocess.check_output', return_value='c3408f34961eb7e8ebc0510306f62238a457dbbc'.encode())
+    @patch("subprocess.check_output", return_value='.release-notes-2.2.0.0.md'.encode())
+    def test_plugin_check(self, *mocks: Any) -> None:
+        component = InputComponentFromSource({"name": "common-utils", "repository": "https://github.com/opensearch-project/common-utils", "ref": "c3408f34961eb7e8ebc0510306f62238a457dbbc"})
+        self.repo = GitRepository(
+            url="https://github.com/opensearch-project/common-utils",
+            ref="c3408f34961eb7e8ebc0510306f62238a457dbbc",
+        )
+        self.assertEqual(self.repo.url, "https://github.com/opensearch-project/common-utils")
+        self.assertEqual(self.repo.ref, "c3408f34961eb7e8ebc0510306f62238a457dbbc")
+        self.assertEqual(self.repo.sha, "c3408f34961eb7e8ebc0510306f62238a457dbbc")
+        release_notes = ComponentsReleaseNotes.from_component(component, "2.2.0", self.repo.dir)
```

Comment:
Test class structure should match the code class structure. So this should be in a `TestComponentReleaseNotesOpenSearch` class and you should be directly creating a `ComponentReleaseNotesPlugin` instead of calling `from_component`. 

The `from_component` method is its own problem, should be in a `TestComponentReleaseNotes` class and have a test that ensures that this method returns the right release notes class based on input.

Reply:
Modifies the tests.